r/R/type.R

```diff
@@ -202,6 +204,9 @@ NestedType <- R6Class("NestedType", inherit = DataType)
 #' either "s" or "ms", while `time64()` can be "us" or "ns". `timestamp()` can
 #' take any of those four values.
 #' @param timezone For `timestamp()`, an optional time zone string.
+#' @param byte_width For `binary()`, an optional integer width to create a
+#' `FixedWidthBinary` type. The default `NULL` results in a `BinaryType` with
```

Comment:
"FixedSizeBinary"?

Reply:
Oh, right, sorry. Guess that felt natural to type since we have `FixedWidthType` as a base class and `byte_width` is the parameter name.

Reply:
```suggestion
#' `FixedSizeBinary` type. The default `NULL` results in a `BinaryType` with
```